recipes/hictk/all/conanfile.py

```diff
@@ -85,6 +85,11 @@ def validate(self):
                 f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
             )
 
+        if self.info.options.get_safe("with_arrow"):
```

Comment:
Just to confirm, this limitation is present in all available versions, right?

Reply:
Yes! All versions that depend on Arrow require `arrow/*:compute=True`.